.github/workflows/acceptance-tests.yml

```diff
@@ -1,7 +1,11 @@
 name: 'Acceptance Tests'
 
 on:
-  workflow_dispatch: {} # workflow can be run manually
+  workflow_dispatch:
+    inputs:
+      terraform_version:
```

Comment:
should we make same change for migration tests?

Reply:
i was thiking about doing in a different PR but it's fine, I'll do it here as it's same idea

Reply:
@AgustinBettati done also for migration tests